src/client/indexform/index.js

```diff
@@ -96,6 +94,29 @@ function IndexForm() {
       });
   };
 
+  const getAllCourses = ({ id }, { value }) => {
+    ltikPromise
+      .then(
+        (ltik) => {
+          axios
+            .post(`/api/forms/getCourses?ltik=${ltik}`, {
+              termCode: id,
```

Comment:
@tatikamsetti Why not just name this offeredTermCode? So it is easily passed to the Registrar API?

Reply:
@tatikamsetti NM, I see in the loop you have a variable called offeredTermCode. So it would conflict..